shaarli_client/main.py

```diff
@@ -0,0 +1,45 @@
+"""shaarli-client main CLI entrypoint"""
+import json
+from argparse import ArgumentParser
+
+from .v1 import ShaarliV1Client
+
+
+def main():
+    """Main CLI entrypoint"""
+    parser = ArgumentParser()
+    parser.add_argument(
+        'endpoint',
+        choices=['info'],
+        default='info',
+        help="REST API endpoint"
+    )
+    parser.add_argument(
+        '-u',
+        '--url',
+        help="Shaarli instance URL"
+    )
+    parser.add_argument(
+        '-s',
+        '--secret',
+        help="API secret"
```

Comment:
It would be a nice improvement to be able to get the secret from a config file, because it's a good practice not to set passwords in command lines, and it would avoid to set it in every client call.